packages/settings-library/src/settings_library/tracing.py

```diff
@@ -1,21 +1,14 @@
-from pydantic import AnyUrl, Field, parse_obj_as
+from pydantic import AnyUrl, Field, conint
 
 from .base import BaseCustomSettings
 
 UNDEFINED_CLIENT_NAME = "undefined-tracing-client-name"
 
 
 class TracingSettings(BaseCustomSettings):
-    TRACING_ZIPKIN_ENDPOINT: AnyUrl = Field(
-        default=parse_obj_as(AnyUrl, "http://jaeger:9411"),
-        description="Zipkin compatible endpoint",
+    TRACING_OPENTELEMETRY_COLLECTOR_ENDPOINT: AnyUrl | None = Field(
+        description="Opentelemetry compatible collector endpoint"
     )
-    TRACING_THRIFT_COMPACT_ENDPOINT: AnyUrl = Field(
-        default=parse_obj_as(AnyUrl, "http://jaeger:5775"),
-        description="accept zipkin.thrift over compact thrift protocol (deprecated, used by legacy clients only)",
-    )
-    TRACING_CLIENT_NAME: str = Field(
-        default=UNDEFINED_CLIENT_NAME,
-        description="Name of the application connecting the tracing service",
-        env=["HOST", "HOSTNAME", "TRACING_CLIENT_NAME"],
+    TRACING_OPENTELEMETRY_COLLECTOR_PORT: conint(ge=1024, le=65535) | None = Field(  # type: ignore
+        description="Opentelemetry compatible collector port"
```

Comment:
Here I still don't get why the tracing settings might be set as None.
What is the typical way of defining it? with an endpoint and a port? or just an endpoint, I guess certainly not with just a port right?

```suggestion
    TRACING_OPENTELEMETRY_COLLECTOR_ENDPOINT: AnyUrl = Field(...,
        description="Opentelemetry compatible collector endpoint"
    )
    TRACING_OPENTELEMETRY_COLLECTOR_PORT: PortInt = Field(..., # or you can put a default port if you wish
        description="Opentelemetry compatible collector port"
```

Doing it this way will also simplify all the checks you do later about whether these are None or not.